01_data-model-and-serialized-rep.adoc

```diff
@@ -1571,12 +1567,22 @@ can have significant performance consequences since the server may need
 to read and serialize all of the data for all of the variables mentioned
 in the DMR even though that data is not transmitted to the client.
 
+For a dmr response the checksum is made visible to the client by the addition
+of an Attribute to each top-level variable in the DMR. This attribute is named
+"`_DAP4_Checksum_CRC32`".
+
 If the request to the server is a data request, then the checksum value
```

Comment:
I would add something like `The attribute DAP4_Checksum_CRC32 is no longer added to the DMR included in the data response. Instead, the client is expected to parse it from the serialized data response and add the _DAP4_Checksum_CRC32 attribute to the parsed DMR`. I think overstating when to expect (and when not), is important IMO.